### PR TITLE
[9.0][subcontracted_service] Various fixes

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This PR contains various fixes that we detected when testing.

Issues we found:
- Was not working together with module 'purchase_request_procurement'. 
- Was not implementing correctly the procurement order methods that are to be used to find the correct rule to use, causing the issue above.
- The view in the company is missing

The following issues remain to be fixed:
- Update the README to explain you need to set the rule in the company
- Perhaps by default the existing companies (demo or default) should have a rule proposed. In our test 'My Company' was blank.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@damdam-s Thanks for your module. It is very interesting. Here some feedback.
cc @eficent

File: subcontracted_service/models/procurement.py
Comment (damdam-s) [inline subcontracted_service/models/procurement.py:12]: not sure we have to keep this decorator.
maybe `@api.multi` with `self.ensure_one()` is better. What do you think ?

**Comment by damdam-s** [comment]
@jbeficent thanks for this PR.

For the remaining issues, there is a method on `res.company` to generate the rules.
Maybe this method has to be called at module install or by whatever script (option choosed here)

File: subcontracted_service/models/procurement.py
Comment (damdam-s) [reply subcontracted_service/models/procurement.py:12]: @jbeficent if it's fine for you. I can do the fix right after merging this one.

**Review by damdam-s — APPROVED** [review]

File: subcontracted_service/models/procurement.py
Comment (JordiBForgeFlow) [reply subcontracted_service/models/procurement.py:12]: Ok, change that after merging.

**Comment by JordiBForgeFlow** [comment]
You can use an init_hook. See https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py#L20

**Comment by JordiBForgeFlow** [comment]
using a post-init hook you can create a default procurement rule for each existing company when the module has been installed.

**Comment by damdam-s** [comment]
@jbeficent yes. I will do that. Thanks for pointing this out !

**Pull request merged**
